nntrainer/layers/layer_node.cpp

```diff
@@ -335,29 +332,46 @@ void LayerNode::finalize() {
  * @brief     Forward Propagation of a layer
  */
 void LayerNode::forwarding(bool training) {
+  if (layerv1)
+    layerv1->forwarding(training);
+      else
```

Comment:
How clang-format formats is quite weird 🤣

Reply:
yeah, i dont know why.